### ISSUE TITLE
Error extracting pandoc's information

### ISSUE THREAD

**Issue by cswingle** [Dahie/caramelize#36]
When I run most `caramelize` functions, including help, I get this error:

```bash
$ caramelize new --trace
Error extracting pandoc's information: undefined method `[]' for nil:NilClass

        @scripting_engine = version_string.match(/Scripting engine: (.+)$/)[1]
                                                                           ^^^
Using made up values instead.
/var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/user_interaction.rb:362:in `method_missing': undefined local variable or method `target_file' for main:Object (NameError)
        from /var/lib/gems/3.1.0/gems/caramelize-1.3.0/bin/caramelize:25:in `block (2 levels) in <top (required)>'
        from /var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/command.rb:187:in `call'
        from /var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/command.rb:157:in `run'
        from /var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/runner.rb:444:in `run_active_command'
        from /var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/runner.rb:58:in `run!'
        from /var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/delegates.rb:18:in `run!'
        from /var/lib/gems/3.1.0/gems/commander-5.0.0/lib/commander/import.rb:7:in `block in <top (required)>'
```

I have pandoc 2.17.1.1:

```bash
pandoc 2.17.1.1
Compiled with pandoc-types 1.22.2.1, texmath 0.12.4, skylighting 0.12.3.1,
citeproc 0.6.0.1, ipynb 0.2
User data directory: /home/cswingley/.local/share/pandoc
Copyright (C) 2006-2022 John MacFarlane. Web:  https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
```

This is Debian bookworm, Linux x86_64.

**Comment by Dahie** [comment]
Hi, thanks for the report. I can confirm the issue and will look into it.

**Comment by Dahie** [comment]
Fix is included in v1.3.1, please update :)
